modules/dr-restoring-cluster-state.adoc

```diff
@@ -561,7 +561,7 @@ When the etcd cluster Operator performs a redeployment, the existing nodes are s
 +
 [source,terminal]
 ----
-$ oc patch etcd/cluster --type=merge -p '\{"spec": {"unsupportedConfigOverrides": null}}
```

Comment:
@jinyunma could you PTAL for QE.

Reply:
/lgtm
GOOD CATCH

Reply:
@geliu2016 should this go into 4.11 only or 4.11-4.15?

Reply:
It's only for 4.11, other docs are okay

Reply:
Yes, only affect 4.11.